testing/badtouch/APKBUILD

```diff
@@ -0,0 +1,35 @@
+# Contributor: kpcyrd <git@rxv.cc>
+# Maintainer: kpcyrd <git@rxv.cc>
+pkgname=badtouch
+pkgver=0.5.1
+pkgrel=0
+pkgdesc="Scriptable network authentication cracker"
+url="https://github.com/kpcyrd/badtouch"
+arch="all"
+license="GPL-3.0"
+depends="ca-certificates"
+makedepends="cargo libcap libressl-dev"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/kpcyrd/$pkgname/archive/v$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
```

Comment:
Since it is the default builddir path it can be ommited.

Reply:
done, thanks!